tests/Fixtures/Test/AbstractFixerTest/UnconfigurableFixer.php

```diff
@@ -0,0 +1,38 @@
+<?php

```

Comment:
@ntzm 
this file is excluded from rules, as all fixtures are, but if not needed, don't use Windows line endings

Reply:
Sorry, forgot to convert it